### PR TITLE
Add github actions CI/CD

### PR DESCRIPTION
**Pull request by mjgpy3**

### CODE REVIEW

File: Yesod/Routes/Flow/Generator.hs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
```
Comment (mjgpy3) [inline Yesod/Routes/Flow/Generator.hs:1]: All changes are lints are restylings.

File: Yesod/Routes/Flow/Generator.hs [hunk 2]
```diff
@@ -14,15 +14,15 @@ module Yesod.Routes.Flow.Generator
...
-import Yesod.Routes.TH.Types
+import           ClassyPrelude         hiding (FilePath)
+import qualified Data.Char             as C
+import qualified Data.List             as L
+import qualified Data.Map              as M
+import           Data.Text             (dropWhileEnd)
+import qualified Data.Text             as T
+import           Filesystem            (createTree, writeTextFile)
+import           Filesystem.Path       (FilePath, directory)
+import           Yesod.Routes.TH.Types
```
Comment (pbrisbin) [inline Yesod/Routes/Flow/Generator.hs:25]: You should copy the configs from megarepo for brittany, stylish, and hlint.

Restyled is doing the default thing without them, which is probably not what we'd want (so much context-sensitive alignment).

Unless we still want to respect upstream's style. If so, we should shut off Restyled altogether.

**Review by pbrisbin — CHANGES_REQUESTED** [review]

File: Yesod/Routes/Flow/Generator.hs (hunk 2 above)
Comment (mjgpy3) [reply Yesod/Routes/Flow/Generator.hs:25]: Will do! Good idea.

**Review by pbrisbin — APPROVED** [review]
Much nicer diff!

**Pull request merged**